corehq/ex-submodules/couchforms/dbaccessors.py

```diff
@@ -126,3 +126,13 @@ def get_commtrack_forms(domain):
         reduce=False,
         include_docs=True
     )
+
+
+def get_exports_by_form(domain):
```

Comment:
same dbaccessors and stale ok comment here. also I think this could have a better name since it's pretty unclear what it's returning.

what would be even better is if we made a proxy class to represent the reduce output from this view so it is clear what the API of the objects returned are rather than just having it have to be know/determined by inspection at runtime or of the couch map/reduce functions